src/app/components/RuntimeEvents/RuntimeEventDetails.tsx

```diff
@@ -279,6 +319,88 @@ export const RuntimeEventDetails: FC<{
           </StyledDescriptionList>
         </div>
       )
+    case RuntimeEventType.consensus_accountsundelegate_start:
+      return (
+        <div>
+          <EventTypeIcon eventType={event.type} eventName={eventName} />
+          <StyledDescriptionList titleWidth={isMobile ? '100px' : '200px'}>
+            <dt>{t('common.from')}</dt>
+            <dd>
+              <AccountLink
+                address={event.body.from}
+                scope={scope}
+                plain={addressSwitchOption !== AddressSwitchOption.Oasis}
+              />
+              {addressSwitchOption === AddressSwitchOption.Oasis && (
+                <CopyToClipboard value={event.body.from} />
+              )}
+            </dd>
+            <dt>{t('common.to')}</dt>
+            <dd>
+              <AccountLink
+                address={event.body.to}
+                scope={scope}
+                plain={addressSwitchOption !== AddressSwitchOption.Oasis}
+              />
+              {addressSwitchOption === AddressSwitchOption.Oasis && <CopyToClipboard value={event.body.to} />}
+            </dd>
+            {/* TODO check is needed because some consensus events temporarily use this for rendering */}
```

Comment:
Not sure what this is about.

Reply:
Leftover from https://github.com/oasisprotocol/explorer/pull/1012/files#diff-63da09cea2b3cda0ceb01e5a7cb74bfd43582ae8b89fcca1e63d5c0adca46514R267. It should be reversed and removed now